src/main/python/main.py

```diff
@@ -0,0 +1,194 @@
+from fbs_runtime.application_context.PyQt5 import ApplicationContext
+import PyQt5.QtWidgets as QtWidgets
+import PyQt5.QtGui as QtGui
+import os
+import sys
+import resources  # noqa: F401
```

Comment:
```suggestion
```
Is this import used?

Reply:
It is needed for loading the font, it runs initialization stuff in Qt.